Gemfile

```diff
@@ -26,7 +26,7 @@ gem 'valid_email'
 gem 'rack-attack'
 gem 'rqrcode'
 gem 'ruby-saml'
-gem 'saml_idp', '~> 0.3.1'
+gem 'saml_idp', '~> 0.3.1', github: 'pkarman/saml_idp', branch: 'principal-asserted-attributes'
```

Comment:
I like that idea. I had referenced the upstream PR here https://github.com/18F/identity-idp/pull/306#issuecomment-235037040 but am hoping it will be merged soonish and might beat this PR to landing.

I will add comment to this branch.